dessia_common/schemas/core.py

```diff
@@ -1042,6 +1067,15 @@ def ellipsis_has_exactly_two_args(self) -> PassedCheck:
             return WrongNumberOfArguments(msg)
         return PassedCheck(f"{self.check_prefix}is not an ill-defined ellipsed tuple : '{self.annotation}'.")
 
+    def get_import_names(self, import_names):
```

Comment:
This is the same for all classes inheriting from TypingProperty and should probably be mutualized there (caution, Union is a bit different, but I think it could be the same. This has to be checked)

Reply:
done !